libs/api_common/windows_platform_common.cpp

```diff
@@ -1,15 +1,16 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
+#include "..\libs\store_helper\user\ebpf_registry_helper_um.h"
```

Comment:
why is this line needed?  Ideally the code in api_common should not know whether the store is the registry or filesystem or whatever else

Reply:
cleaned-up